core/src/messages/agent.rs

```diff
@@ -140,7 +142,7 @@ impl BackendStatsMessage {
 #[derive(Serialize, Deserialize, Debug)]
 pub struct DroneStatusMessage {
     pub drone_id: DroneId,
-    pub cluster: String,
+    pub cluster: ClusterId,
     pub capacity: u32,
```

Comment:
I've been thinking about this capacity idea. Do we get anything out of having the drone report what it thinks its capacity is? rather than deriving it scheduler side from the metrics stream? 

I recall that we thought it might help with the case where a lot of backends spawn and the scheduler overcommits them to a drone based on stale metrics, but isn't that also an issue with the capacity approach?

Reply:
We don't really do anything with this capacity value as of now. I imagine it being surpassed by the stats you're reporting.

I'll leave this in until we decide what capacity-based scheduling looks like, but I agree that it's probably not this.